source/index.html.md

```diff
@@ -626,6 +626,7 @@ curl "https://www.filmmakers.eu/api/v1/talent_agencies/123" \
     "phone": "49613163691950",
     "zipcode": "55122"
   },
+  "filmmakers_url":"https://www.filmmakers.eu/agents/example-agency",
```

Comment:
```suggestion
  "filmmakers_url": "https://www.filmmakers.eu/agents/example-agency",
```

Also please move this up so it's grouped w/ the `homepage_url`

Reply:
Do I have to make any changes to the filmmakers PR to influence the order of the output data?

Reply:
The order is not really relevant as the Api is consumed programmatically, this change here was just for better readability/UX of the docs.